src/resources/user_address/spec/user_address.spec.js

```diff
@@ -21,6 +21,15 @@ const address = {
   country: 'My country',
   zipcode: '123456',
 };
+const addressUpdate = {
+  phone_number: '008812343353',
```

Comment:
I think we made a mistake by having this as a part of the address. Phone number is part of user details not address